packages/vuetify/src/composables/layout.ts

```diff
@@ -115,7 +115,7 @@ const generateLayers = (
 }
 
 // TODO: Remove undefined from layout and overlaps when vue typing for required: true prop is fixed
-export function createLayout (props: { layout?: string[], overlaps?: string[] }) {
+export function createLayout (props: { layout?: string[], overlaps?: string[], fullHeight?: boolean }) {
```

Comment:
- Is `props.layout` even used here?
 - Can't we use `ReturnType<typeof makeLayoutProps>`?